src/types/api.ts

```diff
@@ -167,6 +170,10 @@ export type Mention = {
   display_name: string | null;
   /** Link to the user profile, if applicable */
   link: string | null;
+  /** The user description/bio */
+  description: string | null;
+  /** The date in which the user was created, in ISO format */
+  created: ISODateTime;
```

Comment:
I think the type name here is going to cause confusion here. It is called `Mention` and has a field `created`, but the meaning of that is not "when the _mention_ was created" but rather "when the _subject_ of the mention created their account". Also, user _accounts_ are created, but users "join" Hypothesis rather than being created.

Reply:
I have created a PR to rename this https://github.com/hypothesis/h/pull/9352